website/docusaurus.config.js

```diff
@@ -139,4 +153,38 @@ module.exports = {
       },
     ],
   ],
+  // Misleading API, probably will be refactored in Docusaurus, but this is
+  // simply a list of <link> tags
+  stylesheets: [
+    {
+      rel: 'apple-touch-icon',
+      sizes: '180x180',
+      href: '/ts-node/img/favicon/apple-touch-icon.png',
+    },
+    {
+      rel: 'icon',
+      type: 'image/png',
+      sizes: '32x32',
+      href: '/ts-node/img/favicon/favicon-32x32.png',
+    },
+    {
+      rel: 'icon',
+      type: 'image/png',
+      sizes: '16x16',
+      href: '/ts-node/img/favicon/favicon-16x16.png',
+    },
+    {
+      rel: 'manifest',
+      href: '/ts-node/img/favicon/site.webmanifest',
+    },
+    {
+      rel: 'mask-icon',
+      href: '/ts-node/img/favicon/safari-pinned-tab.svg',
+      color: '#5bbad5',
+    },
+    {
+      rel: 'shortcut icon',
+      href: '/ts-node/img/favicon/favicon.ico',
+    },
+  ],
```

Comment:
This looks new, what does it do?

Reply:
Same as above, just a replacement for the head tags so they show up on every page.